packages/storybook/src/nimble/anchor-tabs/anchor-tabs-matrix.stories.ts

```diff
@@ -36,10 +62,15 @@ const component = (
     widthValue: WidthState
 ): ViewTemplate => html`
     <${anchorTabsTag} activeid="tab1" style="padding: 15px;${widthValue ? ` width: ${widthValue};` : ''}">
-        ${when(() => toolbar, html`
+        ${when(() => toolbar.showToolbar, html`
             <${tabsToolbarTag}>
-                <${buttonTag} appearance="ghost">Toolbar Button</${buttonTag}>
-                <${buttonTag} appearance="ghost" style="margin-left: auto;">Right-aligned Button</${buttonTag}>
+                ${when(() => toolbar.showLeftButton, html`
+                    <${buttonTag} appearance="ghost">Left Button</${buttonTag}>
+                `)}
+                ${when(() => toolbar.showRightButton, html`
+                    <${buttonTag} appearance="ghost">Right Button</${buttonTag}>
+                    <${buttonTag} appearance="ghost">Right Button 2</${buttonTag}>
```

Comment:
Can you double check the matrix stories? these aren't rending on the right (missing slot=end)